src/Spectre.Angular2Client/src/app/app.module.ts

```diff
@@ -26,11 +26,18 @@ import { routing } from './app.routing';
 
 import { AppComponent } from './app.component';
 import { PreparationListComponent } from './preparation-list/preparation-list.component';
-import { PreparationService } from './preparation-list/preparation.service';
+import { PreparationService } from './preparation/preparation.service';
 import { PreparationsModule } from './preparation/preparations.module';
 import { PageNotFoundComponent } from './page-not-found/page-not-found.component';
+import { PlotlyModule } from './plotly/plotly.module';
+import { SpectrumService } from './spectrum/spectrum.service';
```

Comment:
@Sebaxp95 found [this](https://stackoverflow.com/questions/35346342/which-type-of-folder-structure-should-be-used-with-angular-2)